# Imports

In [8]:
import os
import pandas as pd
import matplotlib.pyplot as plt
from IPython.display import display

# Directories

In [2]:
os.chdir("..")
print(os.getcwd())

/home/cotsios/dsit/2nd-semester/ml-in-comp-bio/Assignment-1


In [3]:
DATA_DIR = os.path.join(os.getcwd(), "data")
print(DATA_DIR)

/home/cotsios/dsit/2nd-semester/ml-in-comp-bio/Assignment-1/data


# The data

In [13]:
file_name = "assignment1_dev_set.csv"

df = pd.read_csv(os.path.join(DATA_DIR, file_name))
cols = df.columns

In [14]:
display(cols)

Index(['Unnamed: 0', 'Project ID', 'Experiment type', 'Sex', 'Host age', 'BMI',
       'Disease MESH ID', 'Acholeplasma axanthum',
       'Acidaminococcus fermentans', 'Acidaminococcus intestini',
       ...
       'Clostridium sphenoides', 'Clostridium spiroforme',
       'Clostridium stercorarium', 'Clostridium symbiosum',
       'Clostridium thermosuccinogenes', 'Clostridium xylanolyticum',
       'Eubacterium brachy', 'Eubacterium dolichum', 'Eubacterium sulci',
       'Ruminococcus gnavus'],
      dtype='object', length=141)

We can see that the `BMI` column is the 6<sup>th</sup> one while the bacterial concentrations start from the 8<sup>th</sup> column up to the last (141<sup>st</sup>)  
Also, since we want to classify `BMI` based **only** on the bacteria in the gut microbiome, we can **exclude** the following columns:
- `Project ID`
- `Experiment type`
- `Sex`
- `Host age`
- `Disease MESH ID`